src/engine/strat_engine/backstore/metadata/static_header.rs

```diff
@@ -554,8 +584,8 @@ pub mod tests {
         fn static_header(ref sh1 in static_header_strategy()) {
             let buf = sh1.sigblock_to_buf();
             let sh2 = StaticHeader::sigblock_from_buf(&buf).unwrap().unwrap();
-            prop_assert_eq!(sh1.pool_uuid, sh2.pool_uuid);
-            prop_assert_eq!(sh1.dev_uuid, sh2.dev_uuid);
+            prop_assert_eq!(sh1.identifiers.pool_uuid, sh2.identifiers.pool_uuid);
```

Comment:
Might as well just use one equality check on the identifiers, since they implement ```Eq```.